_data/sites.yml

```diff
@@ -948,6 +948,11 @@
   size: 7.5
   last_checked: 2022-05-05
 
+- domain: eklausmeier.goip.de
+  url: https://eklausmeier.goip.de/
+  size: 27.9
+  last_checked: 2023/05/17
```

Comment:
Not sure how Jekyll handles this date format. Could you replace the slashes with hyphens?